node.gyp

```diff
@@ -441,13 +441,13 @@
             '<(SHARED_INTERMEDIATE_DIR)/node_javascript.cc',
           ],
           'conditions': [
-            [ 'node_use_dtrace=="false" and node_use_etw=="false"', {
+            [ 'node_use_dtrace=="false" and node_use_etw=="false" or node_target_type=="static_library"', {
```

Comment:
Long line.  You can use C-style string concatenation in gyp files:
```python
'a and b or '
'c and d'
# becomes 'a and or c and d'
```

Reply:
I'll go through and fix all these lines. Thanks